docs/tutorials/mcmc_global.ipynb

```diff
@@ -340,18 +340,24 @@
    "outputs": [],
```

Comment:
<p>I <em>think</em> you shouldn't have to specify the path to the data/noisefile here, right? Because they should be used by default if simulate=False and no files are provided?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/21cmfast/21CMMC/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>